biodatasets/citation_gia_test_collection/citation_gia_test_collection.py

```diff
@@ -0,0 +1,342 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+from typing import List
+
+import datasets
+import xml.etree.ElementTree as ET
+import uuid
+import html
+from xml.sax.saxutils import escape, unescape
+
+from utils import schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_CITATION = """\
+@article{Wei2015,
+  doi = {10.1155/2015/918710},
+  url = {https://doi.org/10.1155/2015/918710},
+  year = {2015},
+  publisher = {Hindawi Limited},
+  volume = {2015},
+  pages = {1--7},
+  author = {Chih-Hsuan Wei and Hung-Yu Kao and Zhiyong Lu},
+  title = {{GNormPlus}: An Integrative Approach for Tagging Genes,  Gene Families,  and Protein Domains},
+  journal = {{BioMed} Research International}
+}
+"""
+
+_DATASETNAME = "citation_gia_test_collection"
+
+_DESCRIPTION = """\
```

Comment:
I would put just:
```
The Citation GIA Test Collection was recently created for gene indexing at the NLM and includes 151 PubMed abstracts with both mention-level and document-level annotations. They are selected because both have a focus on human genes.
```

Reply:
Done.